stubs/urllib3/urllib3/connectionpool.pyi

```diff
@@ -1,9 +1,14 @@
-from typing import Any
+from _typeshed import Self
+from collections.abc import Mapping
+from logging import Logger
+from queue import LifoQueue
```

Comment:
This module imports `LifoQueue` from another submodule of `urllib3`, _not_ the stdlib `queue` module

```suggestion
from .util.queue import LifoQueue
```

Reply:
Ah, sorry. That was a diff between their 1.26 and main branch. I had mixed up which tab was which, as it's LifoQueue from `queue` in `main`:
https://github.com/urllib3/urllib3/blob/main/src/urllib3/connectionpool.py#L3

Since `queue` stubs don't exist, I'll add them since the file is only 22 lines:
https://github.com/urllib3/urllib3/blob/1.26.x/src/urllib3/util/queue.py

Reply:
Their `import queue` imports the standard-library module named `queue`. Anything else would have to be `from urllib.util import queue` or `from .util import queue` or something similar. Here are the stubs for the standard-library `queue` module: https://github.com/python/typeshed/blob/master/stdlib/queue.pyi